content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -87,18 +87,18 @@ The three conditions are `ready`, `serving`, and `terminating`.
 
 #### Ready
 
-`ready` is a condition that maps to a Pod's `Ready` condition. A running Pod with the `Ready`
-condition set to `True` should have this EndpointSlice condition also set to `true`. For
-compatibility reasons, `ready` is NEVER `true` when a Pod is terminating. Consumers should refer
+`Ready` is a condition that maps to a Pod's `Ready` condition. A running Pod with the `Ready`
```

Comment:
Checked the source code. For Endpoints, the unofficial condition type is `ready`, for Pod, the official condition type is `Ready`. This inconsistency is annoying, but it is the fact to be documented.

Reply:
> unofficial

But part of Kubernetes, right?

We might want to (separately) sprinkle some note shortcodes to point out the oddness.